smack-extensions/src/main/java/org/jivesoftware/smackx/muc/packet/GroupChatInvitation.java

```diff
@@ -69,6 +69,10 @@ public class GroupChatInvitation implements ExtensionElement {
     public static final QName QNAME = new QName(NAMESPACE, ELEMENT);
 
     private final EntityBareJid roomAddress;
+    private final String reason;
+    private final String password;
+    private final String thread;
+    private final Boolean continue_bool;
```

Comment:
No reason to use a boxed type, also see above regarding snake case.
```suggestion
    private final boolean continue;
```